src/server.rs

```diff
@@ -23,6 +24,17 @@ pub async fn run_server(settings: ServerSettings, tx: mpsc::Sender<model::Report
     Ok(())
 }
 
+/// Healthcheck
+// Returns 200 OK or 503 Service unhealthy for health checking
+async fn health_check()  -> impl IntoResponse  {
+    // TODO consider performing some more sophisticated check here ;)
+    let healthy = true; 
+    match healthy{ 
+        true => (StatusCode::OK, "OK"), 
+        false => (StatusCode::SERVICE_UNAVAILABLE, "Service unhealthy"), 
+    }; 
+}
```

Comment:
I guess it doesn't really matter since the compiler will likely optimise the match away, but I think this shouldn't be included until there's an actual plan for a more sophisticated health check

Reply:
Just wanted to create a skeleton for the future implementation. This service definitively deserves some better check (at least testing if the DB connection works), but for now it would be too much refactoring. What you think would be better, to remove the dummy code completely or just to comment out?

Reply:
I would remove it entirely and just add it back when we know what the future better check looks like